daphne_worker/src/config.rs

```diff
@@ -207,7 +208,7 @@ impl DaphneWorkerConfig {
         let admin_token = match env.secret("DAP_ADMIN_BEARER_TOKEN") {
             Ok(raw) => Some(BearerToken::from(raw.to_string())),
             Err(err) => {
-                console_log!("DAP_ADMIN_BEARER_TOKEN not configured: {:?}", err);
+                error!("DAP_ADMIN_BEARER_TOKEN not configured: {:?}", err);
```

Comment:
Whether this is an error depends on how DaphneWorker is configured. I would suggest `info`.

Reply:
Oh, and we need to initialize tracing for the DOs; I'll add that.  Good catch!

Tracing in the daphne crate will work and I've tested it already, though it's not in this PR.